features/site_configuration.feature

```diff
@@ -161,8 +161,8 @@ Feature: Site configuration
       And I have a post layout that contains "Post Layout: {{ content }} built at {{ page.date | date_to_xmlschema }}"
       And I have an "index.html" page with layout "page" that contains "site index page"
       And I have a configuration file with:
-        | key         | value            |
-        | timezone    | America/New_York |
+        | key         | value        |
+        | timezone    | UTC+04:00    |
```

Comment:
A [thread on StackExchange](http://unix.stackexchange.com/questions/104088/why-does-tz-utc-8-produce-dates-that-are-utc8) for reference..